src/dotnet/commands/dotnet-cache/Program.cs

```diff
@@ -70,13 +71,19 @@ public static CacheCommand FromArgs(string[] args, string msbuildPath = null)
             {
                 msbuildArgs = new List<string>();
 
-                if (string.IsNullOrEmpty(projectArgument.Value()))
+                if (!projectArguments.HasValue())
                 {
                     throw new InvalidOperationException(LocalizableStrings.SpecifyEntries);
                 }
 
                 msbuildArgs.Add("/t:ComposeCache");
-                msbuildArgs.Add(projectArgument.Value());
+                msbuildArgs.Add(projectArguments.Values[0]);
+                var _additionalProjectsargs = projectArguments.Values.Skip(1);
```

Comment:
(nit) the [https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md) say to use `_` prefixes for fields, not local variables.